h/streamer/conf/nginx.conf

```diff
@@ -0,0 +1,63 @@
+daemon off;
```

Comment:
I think trying to use jinja2 somehow to parametrize this files would be useful in the feature. Not sure how to do that without much fuss without moving them to the ansible repo.